docs/contributing/docs-and-website-components.md

```diff
@@ -212,6 +202,14 @@ items={[
 slug={props.slug}
 />
 
+---
+
+## Other available components
```

Comment:
I'm wondering if we should get rid of this section entirely. The remaining components here (Breadcrumb and TableOfContents) aren't really things that you'd import -- they're automatically added to pages in the site layout.

Reply:
Yeah they may not make the most sense in this file compared to other components, but I think someone had pointed out it being good to document what those 2 components were doing since edits to the `doc-links.yaml` and frontmatter that contributors often touch has a `tableOfContentsDepth` and a `breadcrumbTitle`. Recording it here was probably just the best place I found 🤷‍♂️